### ISSUE TITLE
train 

### ISSUE THREAD

**Issue by Arrkwen** [Evergrow/Det_Inpainting#3]
Nice work! Would you consider describe the training process more detailedly, or can you provide the pretraind model (.pth) to test some images? Thanks!

**Comment by Evergrow** [comment]
Thank you for your attention! Please refer [config file](https://github.com/Evergrow/GDN_Inpainting/blob/master/config.yml) to know the meaning of each parameter. We use the current values to train our model. If the model does not converge after 100 epochs, please set epoch larger. To know the detailed training process, you need to watch the loss curve on tensorboard. Below figure is the evaluation loss curve on Celeba-HQ. Both dis_loss and l1_loss are going down, which is the right training way. BTW, the pretrained models were released, please read readme carefully :)
![image](https://user-images.githubusercontent.com/34977573/104080851-83b84c00-5265-11eb-8a76-9bfdf37fa44b.png)

**Comment by noseDewdrop** [comment]
can i get your detailed environment such as numpy、python, some functions has been changed that i can not reproduce these codes. for example, tensorflow have not contrib in over 2 but my python3.7 is not support tensorflow whitch supporting contrib such as 1.12.0.

**Comment by Evergrow** [comment]
I am sorry for the problem you met. This code is based on TensorFlow 1 which is a bit out of style. I will give you two environments, and you can choose the right configuration for your hardware (GPU).


Plan A: for most GPUS such as GTX 1080, RTX 2080, and so on.
- cuda 10.0
- cudnn 7.6.5
- python 3.6
- tensorflow 1.12.0/1.13.1

Plan B: for the latest RTX 30XX. Refer to [this link](https://www.pugetsystems.com/labs/hpc/How-To-Install-TensorFlow-1-15-for-NVIDIA-RTX30-GPUs-without-docker-or-CUDA-install-2005/#UpdateyourNVIDIADriver) to build the environment. 
- cuda 11.1
- cudnn 8.0.4
- python 3.6
- tensorflow 1.15.1

For more information about tensorflow version, please visit [official document](https://www.tensorflow.org/install/source#common_installation_problems). In your case, updating tensorflow version from 1.12.0 to 1.13.1 may work. If this answer cannot solve your trouble, please give me a message.
